src/core_modules/capture-core/components/ScopeSelector/ScopeSelector.types.ts

```diff
@@ -0,0 +1,40 @@
+import type { ReactNode } from 'react';
+
+export type OwnProps = {
+  isUserInteractionInProgress?: boolean;
+  pageToPush?: string;
+  selectedOrgUnitId?: string;
+  selectedProgramId?: string | null;
+  previousOrgUnitId?: string;
+  selectedCategories?: Record<string, any>;
```

Comment:
Change to `selectedCategories?: { [categoryId: string]: { writeAccess: boolean } }`